packages/network-of-terms-reconciliation/test/server.test.ts

```diff
@@ -60,11 +60,11 @@ describe('Server', () => {
     // Results must be sorted by score in decreasing order.
     expect(results.q2.result).toHaveLength(2);
     expect(results.q2.result[0].name).toEqual('All things art');
-    expect(results.q2.result[0].score).toEqual(42.86); // Match of ‘things’ in prefLabel ‘All things art’.
+    expect(results.q2.result[0].score).toEqual(81.65); // Match of ‘things’ in prefLabel ‘All things art’.
     expect(results.q2.result[1].description).toEqual(
       'painted things that can be beautiful • another altLabel'
     ); // Result has no prefLabel.
-    expect(results.q2.result[1].score).toEqual(16.67); // Match of ‘things’ in altLabel ‘painted things that can be beautiful’.
+    expect(results.q2.result[1].score).toEqual(63.25); // Match of ‘things’ in altLabel ‘painted things that can be beautiful’.
```

Comment:
These are quite drastic changes in the scores: a word matching a single word in a much larger string is now scored higher.

Reply:
Yes, indeed! However, I can see the benefits of these new scores: I'd like to give them a try in my test dataset. So: let's merge and find out! :)